community_images/consul/bitnami/configs/acls/consul-anonymous-token-policy.hcl

```diff
@@ -0,0 +1,6 @@
+node_prefix "" {
```

Comment:
where is this file getting used?

Reply:
It'll be used in the docker-compose-acl's consul-server1